src/lib/keys.js

```diff
@@ -53,4 +53,6 @@ export function allKeys( arg ) {
   return arg ? (arg.constructor === Symbol || typeof arg === 'symbol') : Symbol( 'allKeys' );
 }
 
+export const ALL_KEYS = Symbol('ALL_KEYS')
```

Comment:
we have `Symbol( 'allKeys' )` above. could we `export const ALL_KEYS = allKeys()` to dry things up and ensure consistency across? we may need to adjust `store.setBinding` too.